build.cake

```diff
@@ -0,0 +1,198 @@
+#addin "Cake.XdtTransform"
+#addin "Cake.Powershell"
+#addin "Cake.Http"
+#addin "Cake.Json"
+#addin "Newtonsoft.Json"
+
+#load "local:?path=CakeScripts/helper-methods.cake"
+
+
+var target = Argument<string>("Target", "Default");
+var configuration = new Configuration();
+var cakeConsole = new CakeConsole();
+
+var unicornSyncScript = $"./scripts/Unicorn/Sync.ps1";
+
+var configJsonFile = "cake-config.json";
+
+/*===============================================
+================ MAIN TASKS =====================
+===============================================*/
+
+Setup(context =>
+{
+	cakeConsole.ForegroundColor = ConsoleColor.Yellow;
+	
+    var configFile = new FilePath(configJsonFile);
+    configuration = DeserializeJsonFromFile<Configuration>(configFile);
+});
+
+Task("Default")
+.WithCriteria(configuration != null)
+.IsDependentOn("Clean")
+.IsDependentOn("Copy-Sitecore-Lib")
+.IsDependentOn("Publish-Website-Projects")
+.IsDependentOn("Apply-Xml-Transform")
+.IsDependentOn("Modify-Unicorn-Source-Folder")
+.IsDependentOn("Publish-Transforms")
+.IsDependentOn("Sync-Unicorn")
+.IsDependentOn("Deploy-EXM-Campaigns");
+
+Task("Quick-Deploy")
+.WithCriteria(configuration != null)
+.IsDependentOn("Clean")
+.IsDependentOn("Copy-Sitecore-Lib")
+.IsDependentOn("Publish-Website-Projects")
+.IsDependentOn("Apply-Xml-Transform")
+.IsDependentOn("Modify-Unicorn-Source-Folder")
+.IsDependentOn("Publish-Transforms");
+
+Task("Initial")
+.WithCriteria(configuration != null)
+.IsDependentOn("Clean")
+.IsDependentOn("Copy-Sitecore-Lib")
+.IsDependentOn("Publish-Website-Projects")
+.IsDependentOn("Apply-Xml-Transform")
+.IsDependentOn("Modify-Unicorn-Source-Folder")
+.IsDependentOn("Publish-Transforms")
+.IsDependentOn("Initial-From-Sync-Unicorn");
```

Comment:
I believe this issue was resolved by removing the Habitat Home Guide Author from the demoadmin user:

https://github.com/Sitecore/Sitecore.HabitatHome.Content/pull/64
I just merged / fixed today... Would avoid the whole "Initial-From-Sync-Unicorn" thing?

Reply:
Fine, I will list the dependent tasks instead of groupin in "Initial-From-Sync-Unicorn" which will be deleted.

Reply:
Perfect